### ISSUE TITLE
About setting 'Copy Local = true' problem

### ISSUE THREAD

**Issue by kelicto** [DapperLib/Dapper#1691]
Why don't vs copy local Dapper.dll? Could you tell me the reason or how to solve it by a good way.

Reference:
![image](https://user-images.githubusercontent.com/23723575/127143848-7a120a22-1231-4f22-b1ba-52329107640d.png)

**Comment by mgravell** [comment]
If you're asking about transitive dependency resolution and deployment: that only happens when you build an *application*, and I'm guessing that this is a *library*; note also that this would only show in the build output, not the references tree. However, I don't think any of this is Dapper specific.

**Comment by kelicto** [comment]
But `DapperExtension.dll` can be passed by reference resolution.

**Comment by mgravell** [comment]
Ultimately, these are build system questions. Is there something I can help with at the library level?

**Comment by kelicto** [comment]
Yeah. You can write code to copy them or use 'Build Event' to do.
